sql/telemetry_derived/attitudes_daily_v1/query.sql

```diff
@@ -0,0 +1,40 @@
+WITH
+  heartbeat AS (
+    SELECT
+      client_id,
+      payload.survey_id,
+      SPLIT(payload.survey_id, '::')[OFFSET(1)] AS shield_id
+    FROM
+      `moz-fx-data-shared-prod.telemetry_stable.heartbeat_v4`
+    WHERE
+      DATE(submission_timestamp) = @submission_date
+      AND payload.survey_id LIKE 'hb-das%'
+  ),
+  clients_daily AS (
+    SELECT
+      *
+    FROM
+      telemetry.clients_daily
+    WHERE
+      submission_date = @submission_date
+      AND normalized_channel = 'release'
+  ),
+  heartbeat_joined AS (
+    SELECT
+      *
+    FROM
+      `moz-fx-data-shared-prod.external.survey_gizmo_daily_attitudes`
+    JOIN
+      heartbeat
+    USING(shield_id)
```

Comment:
Not that it really matters a ton since this table is tiny, but it *is* partitioned by date and adding a WHERE clause on `date` here will limit the data scanned